### PR TITLE
test: Change RHEL Atomic image to version 7.3

### PR DESCRIPTION
**Pull request by stefwalter**
Blocked on:

 * [x] Upload image

### CODE REVIEW

File: test/images/rhel-atomic [hunk 1]
```diff
@@ -1 +1 @@
-rhel-atomic-d5e0544247f58ab778219aa786d21ecdc610b7a0.qcow2
\ No newline at end of file
+rhel-atomic-67973815e06227393fd42c227f7afaa82a851a2a.qcow2
```
Comment (dperpeet) [inline test/images/rhel-atomic]: Where did you upload this? In your regular image store there is only a recent upload of the previous image to be found.

Comment (dperpeet) [inline test/images/rhel-atomic]: and the test runner can't download the image

Comment (stefwalter) [inline test/images/rhel-atomic]: I must have upload the wrong image.

**Comment by stefwalter** [comment]
Uploaded

**Comment by stefwalter** [comment]
```
error: Failed dependencies:
	libvirt is needed by cockpit-machines-125.x-1.wip.el7.centos.x86_64
	libvirt-client is needed by cockpit-machines-125.x-1.wip.el7.centos.x86_64
```

**Comment by stefwalter** [comment]
Fixed the cockpit-machines issue.

**Comment by stefwalter** [comment]
Needs updates for cockpit-machines, now that they have a dependency.

**Comment by stefwalter** [comment]
And the image is gone again ... will track it down.

**Comment by stefwalter** [comment]
@petervo Could you take a look at what is failing in the testRemoteManagement OSTree case?

**Comment by petervo** [comment]
There no longer seems to be a atomic remote on configured by default. So I just removed the check.

**Pull request closed without merging**